src/sentry/SentryTransactionHub.ts

```diff
@@ -0,0 +1,27 @@
+import * as Sentry from '@sentry/react-native'
+import { Transaction } from '@sentry/types'
+import { SentryTransaction, SentryTransactions } from 'src/sentry/SentryTransactions'
+
+let transactions = [] as Array<Transaction>
+
+export const SentryTransactionHub = {
+  startTransaction(name: SentryTransaction) {
+    const transaction = Sentry.startTransaction({ ...SentryTransactions[name], trimEnd: true })
+    transactions.push(transaction)
+  },
+  finishTransaction(name: SentryTransaction) {
+    // get span operation - 'op'
```

Comment:
this comment is outdated, maybe can be removed :)

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2728/commits/5bdece43e6bdf345c2417fe8757859beb77008aa!